src/store/messagesStore.js

```diff
@@ -1180,6 +1180,9 @@ const actions = {
 						lastActivity: message.timestamp,
 					})
 				}
+				if (message.systemMessage === 'call_ended_everyone') {
+					context.dispatch('setCallHasJustEnded')
```

Comment:
Just to be save, this is only when pulling new messages? And when they are timely near? So not preventing a call when coming back from vacation and scrolling to the end?😅

Reply:
Need to be thoroughly checked now

Reply:
Let's check if current timestamp is within 10 sec after `message.systemMessage` timestamp, we set the flag and timeout duration is `(system message timestamp + 10) - current timestamp` to be precise :p

Reply:
Takes effect only from messages newer than known lastMessageId, so coming back from vacation shouldn't be a problem